### PR TITLE
fix: side navbar rearranged

### LINKED ISSUE #1939: Navigation Drawer: Change sorting, move items, fix design
1. Please change the navigation drawer as follows:
* Instruments
* Logged Data
  * Spacer here
* Connect Device
* Generate Config File
* Settings
  * Spacer here
* About Us
* Buy PSLab -> Link to https://pslab.io/shop/
* FAQ
* Share

2. Move the following items
* Add the link "Feedback and Bugs" to About us (compare to Neurolab)

3. FAQs
* Rename from FAQs to FAQ
* Fix spacing of the actual FAQ screen on the right hand side. There is not spacing to the right of the screen making it difficult to read.
Screenshot_20190815_135151_io pslab

### PR DESCRIPTION
**Pull request by neel1998**
Fixes #1939 

**Changes**: side navbar rearranged

**Screenshot/s for the changes**: 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**:

[sidenav.zip](https://github.com/fossasia/pslab-android/files/3509757/sidenav.zip)

### CODE REVIEW

File: app/src/main/java/io/pslab/fragment/FAQFragment.java [hunk 1]
```diff
@@ -23,13 +23,12 @@ public static FAQFragment newInstance() {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        questions = new String[]{getString(R.string.faq_question1), getString(R.string.faq_question2), getString(R.string.faq_question3), getString(R.string.faq_question4), getString(R.string.faq_question5)};
+        questions = new String[]{getString(R.string.faq_question1), getString(R.string.faq_question2), getString(R.string.faq_question3), getString(R.string.faq_question5)};
```
Comment (CloudyPadmal) [inline app/src/main/java/io/pslab/fragment/FAQFragment.java]: Rename 5 to 4

**Comment by CloudyPadmal** [comment]
I think it would be better to add which flavor the app is to About Us screen. https://riptutorial.com/android-gradle/example/11474/flavor-constants-and-resources-in-build-gradle would help.

File: app/build.gradle [hunk 2]
```diff
@@ -63,9 +63,11 @@ android {
     flavorDimensions 'default'
     productFlavors {
         fdroid {
+            resValue "string", "flavor", "Flavor : fdroid"
```
Comment (CloudyPadmal) [inline app/build.gradle]: Fdroid

File: app/build.gradle [hunk 3]
```diff
@@ -63,9 +63,11 @@ android {
...
             dimension = 'default'
         }
         playstore {
+            resValue "string", "flavor", "Flavor : playstore"
```
Comment (CloudyPadmal) [inline app/build.gradle]: PlayStore

File: app/src/main/java/io/pslab/activity/MainActivity.java [hunk 4]
```diff
@@ -262,22 +260,19 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
...
                         break;
                     case R.id.nav_help_feedback:
-                        navItemIndex = 4;
+                        navItemIndex = 7;
                         CURRENT_TAG = TAG_FAQ;
                         break;
-                    case R.id.nav_report_us:
-                        customTabService.launchUrl("https://goo.gl/forms/sHlmRAPFmzcGQ27u2");
+                    case R.id.nav_buy_pslab:
+                        customTabService.launchUrl("https://pslab.io");
```
Comment (CloudyPadmal) [inline app/src/main/java/io/pslab/activity/MainActivity.java]: https://pslab.io/shop/

File: app/src/main/java/io/pslab/fragment/AboutUsFragment.java [hunk 5]
```diff
@@ -43,6 +44,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         View aboutPage = new AboutPage(getActivity())
                 .isRTL(false)
                 .setImage(R.drawable.logo200x200)
+                .addItem(new Element(getString(R.string.version) + "   " + getString(R.string.flavor), R.drawable.icon_version))
```
Comment (CloudyPadmal) [inline app/src/main/java/io/pslab/fragment/AboutUsFragment.java]: Better add it as a new item. 
If it's PlayStore; > PlayStore version
If it's Fdroid; > Fdroid version

File: app/src/main/res/menu/activity_main_drawer.xml [hunk 6]
```diff
@@ -21,33 +27,29 @@
...
+        android:checkableBehavior="single">
+        <item
+            android:id="@+id/nav_about_us"
+            android:checkable="true"
+            android:icon="@drawable/ic_info_black_24dp"
+            android:title="@string/nav_about_us" />
+        <item
+            android:id="@+id/nav_buy_pslab"
+            android:checkable="true"
+            android:icon="@drawable/ic_buy_icon_black_24dp"
```
Comment (CloudyPadmal) [inline app/src/main/res/menu/activity_main_drawer.xml]: Use <i class="material-icons">shopping_cart</i> icon

File: app/src/main/java/io/pslab/fragment/AboutUsFragment.java (hunk 5 above)
Comment (CloudyPadmal) [reply app/src/main/java/io/pslab/fragment/AboutUsFragment.java]: Use <i class="material-icons">widgets</i> icon

**Comment by mariobehling** [comment]
Please move "Feedback and Bugs" above Version info.

**Comment by mariobehling** [comment]
Use International English!
* No space before ":".
* Write names correctly, e.g. Play store

**Comment by neel1998** [comment]
@mariobehling @CloudyPadmal please review. I have made requested changes

**Comment by neel1998** [comment]
@CloudyPadmal  please merge.

**Pull request merged**